packages/wdio-mocha-framework/src/index.ts

```diff
@@ -159,8 +159,7 @@ class MochaAdapter {
         }
 
         INTERFACES[type].forEach((fnName: string) => {
-            let testCommand = INTERFACES[type][0]
-            const isTest = [testCommand, testCommand + '.only'].includes(fnName)
+            const isTest = TEST_INTERFACES[type].flatMap((testCommand: string) => [testCommand, testCommand + '.only']).includes(fnName)
```

Comment:
This feel like an unnecessary change, you can use the existing setup with the addition of the alias, right?

Reply:
Having just INTERFACES constant doesn't solve this.
https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-mocha-framework/src/index.ts#L162 - this statement makes just 'it' being taken as as a test interface.
I solved this in a similar way it is being done for jasmine. https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-jasmine-framework/src/index.ts#L20 - it has interfaces and test interfaces are two different constants..